src/compiler/binder.rs

```diff
@@ -131,6 +153,18 @@ impl BinderType {
         );
         symbol.set_declarations(declarations);
 
+        if symbol_flags.intersects(
+            SymbolFlags::Class
+                | SymbolFlags::Interface
+                | SymbolFlags::TypeLiteral
+                | SymbolFlags::ObjectLiteral,
+        ) {
+            let mut members = symbol.maybe_members();
+            if members.is_none() {
+                *members = Some(create_symbol_table());
```

Comment:
This pattern feels a little less clunky than in places where I've exposed a separate eg `.set_members()` setter - here, `.maybe_members()` returns a `RefMut<Option<SymbolTable>>`